autoload/vimteractive.vim

```diff
@@ -1,9 +1,57 @@
+let s:iconn_cw_winnr = -1
+
+function! s:show_term(bufnr)
```

Comment:
This is a nice function. Am I correct that this will open up a new split terminal if it has been closed/does not exist in the window (e.g. if you're using tabs)? Could you add a one-line comment explaining this?

Reply:
Yes. My proposed usage for it is `:hide`-ing or `:close`-ing terminal window to free up screen space without losing REPL session.